src/client/components/ProfileFormComponent/ProfileFormComponent.stories.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Avatar from './ProfileFormComponent';
+import avatarImage from '../../assets/images/avatar.png';
```

Comment:
@JyothiNandyala , As i can see Cannot find module '../../assets/images/avatar.png' or its corresponding type declarations. (may be it is a warning)

Reply:
Hi soujanya ,i checked this in other person system's also they are not getting any error like '../../assets/images/avatar.png', may be could you please recheck and let me know.